core/client/ui/radial-menu.js

```diff
@@ -10,12 +11,10 @@ const metaKeyCode = 91;
 const keyToOpen = 'shift';
 const keyToOpenDelay = 200;
 const menuOffset = { x: 0, y: -6 };
-const horizontalMenuItemDistance = { x: 45, y: -90 };
 const radialMenuRadius = 72;
 const radialMenuOffsetY = 38;
 const mouseDistanceToCloseMenu = 105;
-const itemAmountRequiredForBackground = 2;
-const radialMenuStartingAngle = 3.8; // in radians
+const radialMenuStartingAngle = 0; // in radians
```

Comment:
Is this a deliberate change? I think the angle was like that to have a better display when there were less options but it may not be anymore

Reply:
Yes maybe but if you have only 2 or 1, it will have a bad display like this 

https://user-images.githubusercontent.com/18298759/198332017-f741c875-04e3-4b55-937c-c689f7890325.mov